ee/src/main/java/org/jboss/as/ee/subsystem/ManagedExecutorServiceResourceDefinition.java

```diff
@@ -51,6 +53,12 @@
  */
 public class ManagedExecutorServiceResourceDefinition extends SimpleResourceDefinition {
 
+    /**
+     * the resource definition's dynamic runtime capability
+     */
+    public static final RuntimeCapability<Void> CAPABILITY = RuntimeCapability.Builder.of("org.wildfly.ee.concurrent.executor", true, ManagedExecutorServiceAdapter.class)
```

Comment:
Can we not use javax.enterprise.concurrent.ManagedExecutorService for the service type? Generally, we should avoid exposing specific implementation classes via the capability contract.